assets/components/FormSection.tsx

```diff
@@ -1,8 +1,14 @@
 import React, {useState} from 'react';
-import PropTypes from 'prop-types';
 
-export function FormSection({name, testId, children}) {
-    const [opened, setOpened] = useState(name == null);
+interface IProps {
+    initiallyOpen?: boolean;
+    name?: string;
+    testId: string;
+    children: JSX.Element;
+}
+
+export function FormSection({initiallyOpen, name, testId, children}: IProps) {
+    const [opened, setOpened] = useState(initiallyOpen ?? name == null);
```

Comment:
since name was marked in mandatory, probably it doesn't make sense to have a condition where you check if it's `null`. Unless proptypes were wrong.